src/main/resources/edu/cornell/kfs/tax/cu-ojb-tax.xml

```diff
@@ -8,6 +8,7 @@
         <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true" />
         <field-descriptor name="active" column="ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion" />
         <field-descriptor name="lastUpdatedTimestamp" column="LAST_UPDT_TS" jdbc-type="TIMESTAMP" index="true"/>
+        <field-descriptor name="formType" column="FORM_TYPE" jdbc-type="VARCHAR" />

```

Comment:
Please see my note on the related SQL PR. If the "FORM_TYPE" does become part of the primary key, then either this user story or the follow-up user story should update the OJB metadata to configure it as such.

Reply:
I have an open question to the functionals on this

Reply:
Thanks for confirming this with the functionals. Based on the feedback you provided, I'm fine with the form type as-is.